assets/sass/components/sign-in-with-google/_googlesitekit-sign-in-with-google-setup-module.scss

```diff
@@ -18,6 +18,37 @@
 
 .googlesitekit-plugin {
 
+	.googlesitekit-setup-cta-banner__svg-wrapper--sign-in-with-google-setup-cta {
+
+		@media (min-width: $bp-wpdb3) {
+			height: 100%;
+			position: relative;
+		}
+
+		svg {
+			display: block;
+			justify-self: center;
+			max-height: 180px;
+
+			@media (min-width: $bp-nonMobile) {
+				max-height: 210px;
```

Comment:
Tablet fits a little bigger size, otherwise on regular screens 768px it will look disproportional